samples/sensor/ti_hdc/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  name: Nucleo STM32L496ZG TI_HDC example
+tests:
+  test_build:
+    build_only: true
+    platform_whitelist: nucleo_l496zg
```

Comment:
I wonder about the filter here. I think this will (should) compile with any board supporting I2C.
So maybe a `depends on: i2c` would be better here

Reply:
Indeed, this sample should not be bound to Nucleo STM32L496ZG.